typescript/src/adjustAutoImports.ts

```diff
@@ -66,7 +66,7 @@ export function parseIgnoreSpec(spec: string): ParsedIgnoreSetting {
 }
 
 export const getIgnoreAutoImportSetting = (c: GetConfig) => {
-    return c('suggestions.ignoreAutoImports').map(spec => parseIgnoreSpec(spec))
+    return c('suggestions.ignoreAutoImports')?.map(spec => parseIgnoreSpec(spec))
```

Comment:
I had `reading properties of undefined` when started Volar debugger with `--disable-extensions`. I think these changes may make sense.

Reply:
I would investigate this. https://github.com/zardoy/typescript-vscode-plugins/blob/develop/typescript/src/volarConfig.ts you see that plugin activates only after full configuration is received.
Btw could you also remove L26-28 in this or in this file?

Reply:
@maIIady I'm from mobile cannot do it now. Can you do it? This useless log really annoys me

Reply:
A moment